src/test/java/uk/gov/hmcts/darts/common/service/impl/DataAnonymisationServiceImplTest.java

```diff
@@ -95,27 +105,39 @@ void eventEntityAnonymiseNotUpdatedAsNotAllCasesExpiredAndOnlyAnonymiseIfAllCase
 
         UserAccountEntity userAccount = new UserAccountEntity();
         when(eventService.allAssociatedCasesAnonymised(eventEntity)).thenReturn(false);
-        dataAnonymisationService.anonymiseEventEntity(userAccount, eventEntity, true);
+        dataAnonymisationService.anonymiseEventEntity(userAccount, eventEntity, true, false);
         assertThat(eventEntity.getEventText()).isEqualTo("event text");
         assertThat(eventEntity.isDataAnonymised()).isFalse();
         verify(eventService).allAssociatedCasesAnonymised(eventEntity);
         verify(eventService, never()).saveEvent(eventEntity);
     }
 
-    @Test
-    void positiveEventEntityAnonymiseUpdatedAsAllCasesExpiredAndOnlyAnonymiseIfAllCasesExpiredIsTrue() {
+    @ParameterizedTest(name = "Event should be anonymised if all assocaited cases are anonymised. (isManuallyRequested = {0})")
+    @ValueSource(booleans = {true, false})
+    void eventEntityAnonymiseUpdatedAsAllCasesExpiredAndOnlyAnonymiseIfAllCasesExpiredIsTrue(boolean isManuallyRequested) {
```

Comment:
Can you amend the name to use the new convention please

Reply:
Updated